### PR TITLE
Update wireshark-dev.rb

### PR DESCRIPTION
**Pull request by vkz**
Change to version 1.99.2 that offers native OSX support. No X11 required.

### CODE REVIEW

File: Casks/wireshark-dev.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask :v1 => 'wireshark-dev' do
-  version '1.99.1'
-  sha256 '493d4fc48c60898f6bced96eeb0b1e340c9ca28b07bbced04ed69e553f16713f'
-
-  url "http://www.wireshark.org/download/osx/Wireshark%20#{version}%20Intel%2064.dmg"
+  version '1.99.2'
+  sha256 '8d05f1cdca29369f31762380fb655fe4cb837ad860b18ef6bf0384499a9e7104'
+
```
Comment (vitorgalvao) [inline Casks/wireshark-dev.rb]: Please remove this whitespace (not the line, just the extra spaces).

**Comment by fanquake** [comment]
Can you remove the 2.na.dl. from the URL. Then squash your commits.

**Comment by vkz** [comment]
I could, but it doesn't work without that part. I tried it first time I wrote it and tried it again now.

**Comment by fanquake** [comment]
@vkz Ok. Can you squash your commits, and I'll merge this.

**Comment by vkz** [comment]
@fanquake done, i think

**Comment by fanquake** [comment]
@vkz Thanks

**Pull request merged**